addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1115,6 +1115,13 @@ registry.FooterSlideout = publicWidget.Widget.extend({
         this.__pixelEl.style.width = `1px`;
         this.__pixelEl.style.height = `1px`;
         this.__pixelEl.style.marginTop = `-1px`;
+        // On safari, add a background attachment fixed to fix the glitches that
+        // appear when scrolling the page with a footer slide out.
+        if (this.el.classList.contains("o_safari_browser")) {
+            this.__pixelEl.style.backgroundColor = "transparent";
+            this.__pixelEl.style.backgroundAttachment = "fixed";
+            this.__pixelEl.style.backgroundImage = "url(/website/static/src/img/website_logo.png)";
+        }
```

Comment:
I think we should go with this solution. it is the safest IMO and it's better to not disable the effect for safari users :+1: . It's a hack but it's not the only one that exists to make the footer slideout work on different browsers like [this commit](https://github.com/odoo/odoo/commit/c8fc4a113256db92acd1991280d130c1f0fb628e). If you agree that it's the best solution, let's go for it :rocket: . I think it's worth providing a longer explanation in the comment and in the commit message since we'll be adding a hack (and in the future people will wonder why these strange lines of code are there :exploding_head: ). If you can do that and tripple check that it works well on safari with all the footer templates / options. Then I'll ask for access to the browser stack to do the final depp testing. Thanks a lot !